es/ponyfill.js

```diff
@@ -1,17 +1,23 @@
+
+const observableSymbols = new WeakMap();
```

Comment:
Does the following simpler module accomplish the same thing without reference to `root` or use of WeakMap?

```js
export default Symbol('observable');
```
where index.js line 18 would just be
```js
import result from 'ponyfill.js';
```

If not, I don't understand what we're trying to accomplish.

Reply:
The `WeakMap` was only necessary to make this work with the unit test, which runs the ponyfill multiple times in the same execution environment. If I were to reframe the tests such that they run the ponyfill only once per process, it would be sufficient to capture the `result` in a module scope singleton.

Reply:
If you want the same reference to a `symbol`, then `Symbol(description)` won't do that, but we don't need the `WeakMap`. We could use `Symbol.for('symbol-observable-ponyfill')` or the like, and have a unique, but shared `symbol`. (So long as `Symbol.for` exists, which it likely does if `Symbol` exists).

Reply:
I know that two evaluations of `Symbol('observable')` will give two separate symbols, which is not what we want. But (under current conditions) a module has one instance, so all importers should get the same symbol.

Yes, using `Symbol.for('observable')` would be a more robust way to ensure that there is only one such symbol.

Reply:
I’ll take another look at this with an eye for removing the `WeakMap` and using `Symbol.for` to ensure idempotence instead.